data/json/snippets/mutant_anatomy.json

```
@@ -158,6 +160,54 @@
       "As you cut into its flesh you are disturbed by its sickly-sweet smell, like fermenting fruit"
     ]
   },
+  {
+    "type": "snippet",
+    "category": "<bug_harvest_centipede_chitin>",
+    "text": [
+      "Its legs contain no flesh, just sinews hanging in yellow-green ichor",
+      "The legs fall off easily, leaving the main body looking like an armored worm",
+      "Each pair of legs is slightly different.  These end in hooks, the next in talons like a bird's, there's a pair of chitinous tentacles",
+      "The mandibles keep snapping, and acrid-smelling venom wets the soil.  You decide to begin your work at its tail",
+      "You're startled as its antenna whips out and coils on your arm, if only for a moment",
```

Is the startlement or the coiling only for a moment?

---

The coiling, but that's a sentence I'm not very happy with.

---

```suggestion
      "You're startled as its antenna whips out and momentarily coils on your arm",
```
Briefly or another near-synonym might be better.